packages/p2p/src/components/advertiser-page/advertiser-page.scss

```diff
@@ -3,6 +3,62 @@
     display: flex;
     flex-direction: column;
 
+    &__dropdown {
+        box-shadow: 0 1rem 2rem var(--shadow-menu);
+        border-radius: $BORDER_RADIUS;
+        cursor: pointer;
+        position: absolute;
+        right: 0;
+
+        .dc-dropdown-container {
+            margin-top: 0;
+
+            .dc-dropdown__display-placeholder-text {
+                color: var(--text-prominent);
+                background-color: var(--general-main-2);
+            }
+
+            .dc-dropdown__display {
```

Comment:
Dropdown component has all sorts of classNames, we can use those instead of overriding the component's class.